### ISSUE TITLE
Unbuffered comments

### ISSUE THREAD

**Issue by mrmlnc** [static-dev/html-whitespace-parser#16]
## Problem

Support for unbuffered comments.

#### buffered comments

```jade
// comment
p text
```

```html
<!-- comment -->
<p>text</p>
```

#### unbuffered comments

```jade
//- comment
p text
```

```html
<p>text</p>
```

/cc @jescalan for discussion 🌮

**Comment by jescalan** [comment]
Yeah I just haven't added this feature yet, haven't had a need for it. It shouldn't be too tough to add though!

**Comment by mrmlnc** [comment]
Closed by #17.
